### DISCUSSION TITLE
Make all bars toggable

### DISCUSSION THREAD

**Discussion by svennidal** [Kong/insomnia#5580] [Ideas]
Would be nice to be able to hide the project sidebar, the top bar, and the bottom bar. One of the strongest features of Insomnia is the low clutter UI. Except for these 3 bars that remind me of the internet explorer toolbar era.

Would be awesome to be able to toggle these bars, that are just making the workspace on a small monitor even smaller.

I'm sorry if there is already a way, but I couldn't find any.

![insomnia_cluttered_view02](https://user-images.githubusercontent.com/5677980/209304620-c3a9efd4-18b7-4234-86b4-8720fce7dc25.png)
![insomnia_cluttered_view01](https://user-images.githubusercontent.com/5677980/209304624-2e224439-0747-4153-9e66-b5b930f89f82.png)

**Comment by Xanvial** [comment]
This is one of my main reason moved from Postman to Insomnia, wasted UI space for something that I think won't be used much (personally, won't be changing project type or preference a lot in a single working session)

Also this new UI for Projects is kinda unused considering creating a new Projects is hidden behind account login
Other option is to move this to the header together with Projects/Collection path

**Comment by subnetmarco** [comment]
The organization sidebar will be allowed to be toggled next week in 9.0 with a toggle in the footer.

**Before:**

![image](https://github.com/Kong/insomnia/assets/964813/aa20ea3c-5aa5-44e2-9c80-4a0319d077d4)

**After:**

![image](https://github.com/Kong/insomnia/assets/964813/8d2facf3-cca3-43d4-a89e-6bf0ad8be5ba)

**Reply by DusanLesan** [reply]
That looks nicer, but can we please get options to hide header and footer UI as well? I do not ever need them and if I need settings, I press alt for menu bar. Also for those that need that but rarely, you can map toggle to some key. 

I have overridden some css to get that behavior, but it would be better to have that officially supported
<details><summary>Image with my changes</summary>
<p>

![DP-2-20240417-213624](https://github.com/Kong/insomnia/assets/20520590/d48133bc-a22c-415d-aaf9-4bb5e088dd9e)


</p>
</details>

**Reply by keiichi000** [reply]
@DusanLesan - Are you able to publish your theme file that makes Insomnia look like this. This is 1000% better than any of the available layouts, and is exactly the style of layout I have been looking for.

**Reply by DusanLesan** [reply]
@keiichi000 I have published it now: https://www.npmjs.com/package/insomnia-plugin-vscode-theme-minimal


In case it does not work, I have it in my dotfiles repo so you can copy paste: https://github.com/DusanLesan/dotfiles/tree/master/.config/Insomnia/plugins/insomnia-plugin-vscode-theme.


I am thankful that it is possible to modify the UI in such a way but it would be nice for them to do it officially. Here is some pool related to it: https://github.com/Kong/insomnia/discussions/7468

**Reply by keiichi000** [reply]
@DusanLesan - Thank you very much!  Your effort is greatly appreciated and I agree that it would be so much better if there was official support for customizing the layouts like this.
